src/psyclone/domain/gocean/transformations/gocean_opencl_trans.py

```diff
@@ -309,11 +309,11 @@ def _insert_ocl_arg_setter_routine(node, kernel):
         argsetter.symbol_table.add(check_status)
 
         # All arguments are read-only, create the interface for later use
```

Comment:
Please could you remove this comment as it's no longer relevant.

Reply:
Ups, of course, removed now.